.github/ci-gradle.properties

```diff
@@ -19,7 +19,8 @@ org.gradle.parallel=true
 org.gradle.workers.max=2
 
 kotlin.incremental=false
-kotlin.compiler.execution.strategy=in-process
+# Commented for avoid OOMs due to metaspace usage in KSP with Kotlin 1.9.0
+# kotlin.compiler.execution.strategy=in-process
```

Comment:
This can be entirely removed (see https://github.com/android/nowinandroid/pull/933#issuecomment-1804041711)
```suggestion
```